kratos/includes/key_hash.h

```diff
@@ -50,22 +56,21 @@ namespace Kratos
     }
     
     /**
-     * This method combines hash until it reaches the last class in order to obtain a corresponding seed
-     * @param TClassType The type of class to be hashed
+     * @brief This method combines hash until it reaches the last class in order to obtain a corresponding seed
+     * @tparam TClassType The type of class to be hashed
      * @param First The first class to be compared
      * @param Last The last class to be compared
      * @return The resulting seed
      */
     template <class TClassType>
-    inline std::size_t HashRange(
+    inline HasType HashRange(
```

Comment:
I think it should be HashType not HasType

Reply:
True